alf/networks/encoding_networks.py

```diff
@@ -405,12 +440,19 @@ def __init__(self,
             self._state_spec.append(self._create_lstm_cell_state_spec(hs))
             input_size = hs
 
+            last_activation = (activation
+                               if last_activation is None else last_activation)
+            last_kernel_initializer = (kernel_initializer
+                                       if last_kernel_initializer is None else
+                                       last_kernel_initializer)
+
```

Comment:
generally we can directly pass these two arguments which will be later handled by EncodingNetwork

Reply:
removed these lines.